src/easylink/graph_components.py

```diff
@@ -42,63 +48,38 @@ class InputSlot:
     env_var: str | None
     """The environment variable that is used to pass a list  of data filepaths to 
     an ``Implementation``."""
-    validator: Callable[[str], None] = field(compare=False)
+    validator: Callable[[str], None] | None = field(compare=False)
```

Comment:
validators can now be None b/c of SplitterSteps (we don't validate the files coming into the splitter step, but instead continue validating them (now chunks) going into the actual implemented step).